experimentation/src/main/java/com/automattic/android/experimentation/remote/AssignmentsDtoMapper.kt

```diff
@@ -17,4 +17,15 @@ internal object AssignmentsDtoMapper {
             fetchedAt = fetchedAt,
         )
     }
+    fun Assignments.toDto(): Pair<AssignmentsDto, Long> {
+        return AssignmentsDto(
+            variations = variations.mapValues { (_, value) ->
+                when (value) {
+                    is Variation.Control -> null
```

Comment:
Question (❓): Similar to the previous question, why did you chose to map this to `null` and not `control`? I understand that the information is lost anyway as `Variation.Control` can mean either one, but I think using `control` explicitly, even in cases of `null` feels more appropriate to me. Wdyt?

PS: Having said that, I personally think that saving the `AssignmentsDto` instead of `Assignments` might be the best, to avoid boilerplate, or losing information (no matter its importance) and match the [FluxC](https://github.com/wordpress-mobile/WordPress-FluxC-Android) implementation, just to be on the safest side.

Reply:
> even in cases of null feels more appropriate to me. Wdyt?

I don't know - I think I don't share similar feeling, but I can apply such change if it makes things more readable.

Knowing we shouldn't base the internal-persistence on API, just sharing that API uses `null` to annotate control group, at least in these experiments I was checking.

Reply:
Addressed in 0c1b7bf

Reply:
I was thinking that the API is using `null` on rare cases, and instead is usually using `control`, but if that's not the case, then maybe using `null` is actually better, even if that doesn't make this more readable, not sure... After we've discussed this, I'm living this decision up to you. 👍 

PS: Btw, can we write a quite comment on `null` vs. `control` somewhere so that we at least are aware of the differences, usages or each, maybe this will help us in the future, or help other devs like us. Wdyt? 🤔

Reply:
> Addressed in https://github.com/Automattic/Automattic-Tracks-Android/commit/0c1b7bf0db499381f8bde0724450bad274702f4c

Good call on using a constant, actually, we should have used a constant from the start! 🥇

Reply:
Sure, added comment as well!

Reply:
Noted, thanks so far, I think we can merge this now! 🚀